packages/protocol/contracts/governance/ReleaseGoldInstance.sol

```diff
@@ -254,6 +256,16 @@ contract ReleaseGoldInstance is UsingRegistry, ReentrancyGuard, IReleaseGoldInst
     emit DistributionLimitSet(beneficiary, maxDistribution);
   }
 
+  /**
+   * @notice Sets the beneficiary of the instance
+   * @param newBeneficiary The address of the new beneficiary
+   */
+  function setBeneficiary(address payable newBeneficiary) external onlyBeneficiary {
```

Comment:
Similarly, should we allow updating of the refundAddress?

Reply:
Ideally, probably yes, though the likelihood of key rotation for that refund address seem to be much lower @asaj